README.rst

```diff
@@ -184,20 +190,28 @@ keys:
     ``travis``
         Configuration for retrieving logs from Travis-CI.com.  Subfield:
 
-        ``path``
-            *(required)* A template string that will be instantiated for each
-            job of each build to produce the path for the file (relative to the
-            current working directory) in which the job's logs will be saved.
-            See "`Path Templates`_" for more information.
+        ``paths``
+            *(required)* A mapping giving `template strings <Path Templates_>`_
+            for the paths at which to save various types of assets.  Subfield:
+
+            ``logs``
+                A template string that will be instantiated for each job of
+                each build to produce the path for the file (relative to the
+                current working directory) in which the job's logs will be
+                saved.  If this is not specified, no logs will be downloaded.
 
     ``appveyor``
         Configuration for retrieving logs from Appveyor.  Subfields:
 
-        ``path``
-            *(required)* A template string that will be instantiated for each
-            job of each build to produce the path for the file (relative to the
-            current working directory) in which the job's logs will be saved.
-            See "`Path Templates`_" for more information.
+        ``paths``
+            *(required)* A mapping giving `template strings <Path Templates_>`_
```

Comment:
It isn't required any longer or is it?

Reply:
It is required, but it can be empty.

Reply:
would making it truly optional be doable?  it is kinda odd to allow for it to be empty as indicator of "optional"

Reply:
It's now optional.